app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
+  # belongs_to :user, class_name: 'user'
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
   belongs_to :user, class_name: 'User'
```

Comment:
- [ ] A post belongs to the author, kindly consider using author instead of user e.g `belongs_to :author, class_name: 'User'
`

Reply:
Dear Ismailco,

Thank you for the feedback, However, my migration uses users and not author. This makes more sense as it reflects the relationship between the user and and post. author does not work for me during migration.